modules/standard/raid/loot_lists_controller.py

```diff
@@ -115,11 +123,41 @@ def apf_cmd(self, _, category):
         else:
             return "No loot registered for <highlight>%s<end>." % category
 
+    @command(command="albtraum", params=[Options(["c&cm", "pbc", "r&pu", "Ancients", "Samples"], is_optional=True)],
+             description="Get list of items from Albtraum", access_level="all")
+    def albtraum_cmd(self, _, category):
+        if category is None:
+            blob = ""
+            sql = "SELECT category FROM raid_loot WHERE raid = 'Albtraum' GROUP BY category"
+            raids = self.db.query(sql)
+
+            for raid in raids:
+                add_loot = self.text.make_chatcmd("Add loot", "/tell <myname> loot addraid Albtraum %s" % raid.category)
+                show_loot = self.text.make_chatcmd(
+                    "Loot table", "/tell <myname> albtraum %s" % self.get_real_category_name(raid.category, True))
+
+                sql = "SELECT COUNT(*) AS count FROM raid_loot WHERE category = ?"
+                count = self.db.query_single(sql, [raid.category]).count
+
+                blob += "%s - %s items\n" % (raid.category, count)
+                blob += " | [%s] [%s]\n\n" % (show_loot, add_loot)
+
+            return ChatBlob("Albtraum loot tables", blob)
+
+        category = self.get_real_category_name(category)
+
+        items = self.get_items("Albtraum", category)
+
+        if items:
+            return ChatBlob("%s loot table" % category, self.build_list(items, "Albtraum", category))
+        else:
+            return "No loot registered for <highlight>%s<end>." % category
+
```

Comment:
this should be two separate command handlers, one with the category and one without, since they do different things.  see: https://github.com/Budabot/Tyrbot/wiki/Module-Development-Guidelines#optional-command-params

Reply:
I'm not sure if i get it right, sry. :)

Do you want to say, that i should extract the !albtraum command from the handler "!albtraum [...]"?
-> cmd "!albtraum"
-> cmd "!albtraum <....>"
instead of "!albtraum [...]"

if so, should i also do it for the other cmd*s present in the module?

Reply:
yes, correct.

this module was written before I had created the best practices for module develop, but yes the rest of the commands in this controller should be split up as well.  but that doesn't have to be done in this PR since those commands are already written.

Reply:
okay, ty. :) working at it.